nservicebus/msmq/routing_content_core_[6,).partial.md

```diff
@@ -11,6 +11,8 @@ NOTE: When running under ASP.NET the `instance-mapping.xml` file may not be loca
 
 snippet: InstanceMappingFile
 
+NOTE: The endpoint names are case sensitive e.g. if endpoint name is used in routing API and the instance mapping file, the file entries will be used only when there is a case sensitive match. 
```

Comment:
I would lean to keeping the example part. This doco [page](https://docs.particular.net/nservicebus/msmq/routing) shows usage of endpoint names only in xml file (which uses each name once) so it might be confusing for readers what is matched with what.